src/main/kotlin/com/google/androidstudiopoet/input/BuildSystemConfig.kt

```diff
@@ -0,0 +1,3 @@
+package com.google.androidstudiopoet.input
+
+data class BuildSystemConfig(var version: String?, var agpVersion: String?,  var kotlinVersion: String?)
```

Comment:
I'm not sure about that, because I hope that we'll introduce other build systems as well, I can change it to `buildSystemVersion`, would that work for you?

Reply:
yes